inspirehep/modules/records/jsonschemas/records/authors.json

```diff
@@ -86,23 +86,11 @@
                         "description": "FIXME: this is redundant?",
                         "format": "email"
                     },
-                    "rank": {
-                        "enum": [
-                            "SENIOR",
-                            "JUNIOR",
-                            "STAFF",
-                            "VISITOR",
-                            "PD",
-                            "PHD",
-                            "PhD",
-                            "MAS",
-                            "UG",
-                            "Diploma",
-                            "Bachelor",
-                            "smirnov@ictp.it"
-                        ],
-                        "type": "string"
+                    "_rank": {
```

Comment:
I like this idea of using `_field` to hold the raw value of the field. I don't think we are doing this consistently, though.